superset/config.py

```diff
@@ -87,6 +87,11 @@
 # },
 FAVICONS = [{"href": "/static/assets/images/favicon.png"}]
 
+FEATURE_FLAGS = {
```

Comment:
Do you need this change? these lines will set those features enabled for all users.

Reply:
yes

Reply:
Another way to do it,set the value in DEFAULT_FEATURE_FLAGS. "DASHBOARD_CROSS_FILTERS": True

Reply:
Not necessary,you can set the value in DEFAULT_FEATURE_FLAGS. eg. "DASHBOARD_CROSS_FILTERS": True

Reply:
I mean these flags are not necessary for fixing filter mapping issue, and will cause other users to enable these features. So you should remove them from this PR.

Reply:
but...if you set the param to TRUE,the problem will present...if you close it,that's ok.....

Reply:
you can list the necessary feature flags in the description, like `Steps to reproduce this issue`. Other people, like QA or reviewers, read your description and reproduce your issue in their local development environment.

But you should not commit these feature flags in the PR. Once it is merged, it might impact all the users in the Superset community.

Reply:
ok